telemetry/core/src/main/scala/io/buoyant/telemetry/TelemeterInitializer.scala

```diff
@@ -13,7 +13,7 @@ trait TelemeterInitializer extends ConfigInitializer {
 }
 
 @JsonTypeInfo(
-  use = JsonTypeInfo.Id.NAME,
+  use = JsonTypeInfo.Id.CLASS,
```

Comment:
I was not able to get it to work with id. We can think about improving this once we have v1 deployed.